core/src/UserSettingsDialog.svelte

```diff
@@ -305,19 +307,174 @@
     }
   }
 </script>
 
+<svelte:window on:resize={onResize} on:keydown={handleKeyDown} />
+<div class="fd-dialog fd-dialog--active lui-usersettings-dialog" tabindex="0">
+  <div
+    class="fd-dialog__content lui-usersettings-dialog-size"
+    role="dialog"
+    aria-modal="true"
+    aria-labelledby="dialog-title-2"
+  >
+    <div class="fd-dialog__body lui-usersettings-body">
+      <div class="lui-usersettings-left-nav">
+        <div class="fd-side-nav">
+          <div class="fd-side-nav__group-header">
+            <h2 class="fd-title fd-title--h5" id="dialog-title-2">
+              {$getTranslation(dialogHeader)}
+            </h2>
+          </div>
+          <div
+            class="fd-side-nav__main-navigation lui-fd-side-nav__main-navigation"
+          >
+            <ul
+              class="fd-list fd-list--byline fd-list--navigation lui-us-list"
+              role="list"
+            >
+              {#each Object.entries(userSettingGroups) as [key, userSettingGroup], index}
+                {#each Object.entries(userSettingGroup) as userSettingsGroupProperty}
+                  <li
+                    role="listitem"
+                    class="fd-list__item fd-list__item--link lui-us-navlist__item"
+                    data-testid="us-navigation-item"
+                    on:click|preventDefault={() =>
+                      openEditor(userSettingsGroupProperty, event)}
+                    on:keydown={event => handleKeyUp(event, [index])}
+                    tabindex="0"
+                  >
+                    <a tabindex="-1" class="fd-list__link" href="#">
+                      {#if userSettingsGroupProperty[1].icon}
+                        {#if hasOpenUIicon(userSettingsGroupProperty[1])}
+                          <span class="fd-list__thumbnail">
+                            <i
+                              role="presentation"
+                              class={getSapIconStr(
+                                userSettingsGroupProperty[1].icon
+                              )}
+                            />
+                          </span>
+                        {:else}
+                          <span
+                            class={userSettingsGroupProperty[1]
+                              .iconClassAttribute ||
+                              'fd-image--s fd-list__thumbnail'}
+                            aria-label={userSettingsGroupProperty[1].altText
+                              ? userSettingsGroupProperty[1].altText
+                              : ''}
+                            style="background-image:url('{userSettingsGroupProperty[1]
+                              .icon}'); background-size:cover;"
+                          />
+                        {/if}
+                      {:else if userSettingsGroupProperty[1].initials}
+                        <span
+                          class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--thumbnail fd-user-menu__avatar"
```

Comment:
`margin-right: 0.75rem;` is missing
![Bildschirmfoto 2022-02-21 um 19 31 16](https://user-images.githubusercontent.com/14272184/155010764-ea8f2694-8b43-4577-9a32-af64fa008108.png)